### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,16 @@
...
-this plugin you receive nested facts from chef-client.
+this plugin you receive nested facts from chef-client. You can find all information 
+[in plugin manual](http://www.theforeman.org/plugins/foreman_chef/0.1)
 
 # Installation
 
-If you want to use this in production I recommend to combine this plugin with foreman-background
-(https://github.com/ohadlevy/foreman-background) which runs report and facts import as a background
-tasks.
+There are two ways to install the plugin.
```
Comment (iNecas) [inline README.md]: Nitpick: `There are two ways to install the plugin:` Long live bikeshedding! :) ACK

File: README.md [hunk 2]
```diff
@@ -2,17 +2,16 @@
 
 This plugin adds a Chef fact importer to Foreman. It basically means that when you setup your chef
 clients to use foreman handlers (https://github.com/theforeman/chef-handler-foreman) and install
-this plugin you receive nested facts from chef-client.
+this plugin you receive nested facts from chef-client. You can find all information
```
Comment (mmoll) [inline README.md]: in _the_ plugin manual

**Comment by ares** [comment]
both fixed, would you guys mind rereviewing? :-P

**Comment by mmoll** [comment]
:+1:

**Comment by iNecas** [comment]
:+1:

**Comment by ares** [comment]
Thanks for reviews!

**Pull request merged**